WilmervH.md

```diff
@@ -0,0 +1,60 @@
+| Wire connectors              | [AliExpress](https://www.aliexpress.com/item/4000394171442.html?spm=a2g0o.productlist.main.59.78f43ce6rZsG19&algo_pvid=af3fc2d3-ce42-4fe5-9b60-738232f0f8d9&aem_p4p_detail=202212231358062303587920410400021493881&algo_exp_id=af3fc2d3-ce42-4fe5-9b60-738232f0f8d9-29&pdp_ext_f=%7B%22sku_id%22%3A%2210000015036663976%22%7D&pdp_npi=2%40dis%21EUR%210.49%210.4%21%21%21%21%21%40211bda9b16718326862433310d0725%2110000015036663976%21sea&curPageLogUid=Ng4X7oH1Iqnw&ad_pvid=202212231358062303587920410400021493881_8&ad_pvid=202212231358062303587920410400021493881_8) | 2,40 €          |
+| AliExpress shipping          | N/A                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | 12,11 €         |
+|                              |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |                 |
+| 2x Raspberry pi              | Dependant on availability                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | up to 200 euros |
```

Comment:
In order for Hack Club to give you money to purchase these, you'll need to provide a link to where you might find them. https://rpilocator.com/ may be helpful if you can find your country.